.gitattributes

```diff
@@ -0,0 +1,9 @@
+build/*          export-ignore
+docs/*           export-ignore
+tests/*          export-ignore
+.editorconfig    export-ignore
+.gitattributes   export-ignore
+.gitignore       export-ignore
+.travis          export-ignore
```

Comment:
There is no `.travis` file, only `.travis.yml` – is this a bug or just `.travis` consumed as file prefix, so `.travis.yml` suits this pattern?